### PR TITLE
Added preverify.MaxContentLength.

### PR DESCRIPTION
**Pull request by yuizumi**

### CODE REVIEW

File: processor/preverify/preverify.go [hunk 1]
```diff
@@ -30,8 +30,18 @@ type Config struct {
 	// nil or empty implies []int{http.StatusOK}, which is considered to be
 	// the current best practice.
 	GoodStatusCodes []int
+
+	// MaxContentLength
+	//
+	// Zero implies DefaultMaxContentLength, and a negative number implies
+	// "unlimited."
```
Comment (tomokinat) [inline processor/preverify/preverify.go]: Might better to also document "negative implies unlimited" behavior in the description  of flagMaxContentLength. Users may sometimes want to run the packager without the size limit.

Comment (yuizumi) [reply processor/preverify/preverify.go]: Good point. Done.

File: cmd/webpackager/flag_packager.go [hunk 2]
```diff
@@ -48,6 +49,9 @@ var (
 	flagCertURL      = flag.String("cert_url", "", `Certficiate chain URL. (required)`)
 	flagPrivateKey   = flag.String("private_key", "", `Private key PEM file. (required)`)
 
+	// Processor
+	flagSizeLimit = flag.String("size_limit", "4194304", `Maximum size of resources allowed for signed exchanges, or "none" to set no limit.`)
```
Comment (tomokinat) [inline cmd/webpackager/flag_packager.go]: optional: Maximum size of resources in bytes

Comment (tomokinat) [reply cmd/webpackager/flag_packager.go]: optional: re-use noSizeLimitString here, instead of writing "none" literally.

Comment (Montana) [reply cmd/webpackager/flag_packager.go]: Thank you!

Comment (yuizumi) [reply cmd/webpackager/flag_packager.go]: Updated the help description. Glad to know it's something really wanted.

**Pull request merged**